### PR TITLE
Support minimal GitHub Pages deployment

### PR DESCRIPTION
**Pull request by f-laurens**
Related to #261.
Add files to support minimal GitHub Pages deployment.
Also slightly cleanup the `README.md` `PROTOCOL.md` and `CONTRIBUTING.md` files that will be visible on https://nabaztag2018.github.io/pynab/.

### CODE REVIEW

**Comment by f-laurens** [comment]
Note: once this PR is merged, GitHub Pages deployment should be activated by the powers in charge, from https://github.com/nabaztag2018/pynab/settings/pages : deploy `/ (root)` from `master` branch.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -125,12 +104,25 @@ If tests fail, fix them and push to the branch, either with new commits or
...
 git fetch origin
 git checkout master
 bash upgrade.sh
 ```
 
-The last command is required to run a full upgrade for changes in localized
+The last command is required to run a full upgrade for any changes in localized
 messages, drivers and dependencies.
+
+## How to test a contributor Pull Request
```
Comment (mehdichaouch) [inline CONTRIBUTING.md:116]: This part is very sad without emoji ! :disappointed:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
-# Welcome to Pynab!
+# Contribution guidelines
```
Comment (mehdichaouch) [inline CONTRIBUTING.md:1]: IMO not very welcoming !

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Nabaztag en Python pour Raspberry Pi
+# [Nabaztag en Python pour Raspberry Pi](https://github.com/nabaztag2018/pynab/releases)
```
Comment (mehdichaouch) [inline README.md:1]: If you add a link, text should reflect on link. Here it's not the case.

`[pynab releases](https://github.com/nabaztag2018/pynab/releases)`

Comment (f-laurens) [reply README.md:1]: > If you add a link, text **should** reflect of link.

Why?
In any case, we should not change the document title  from Nabaztag en _Python pour Raspberry Pi_ to _pynab releases_.
But the link may be removed, if the powers in charge prefer it so.

**Pull request merged**